src/services/FileSystemService.real.spec.ts

```diff
@@ -295,7 +295,7 @@ describe('FileSystemService - REAL', () => {
         expect(result.type).toEqual('file');
       });
 
-      it('it returns metadata for broken Symlink but only as a File', async () => {
+      it.skip('it returns metadata for broken Symlink but only as a File', async () => {
```

Comment:
Add comment whit it is skipped.

Reply:
Fixed in b4c96e4c7244afcd90bac9258a1241bb26cc4304